src/tests/helpers/intervalDigest.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * 	INFO: dirty hack to over come angular-co wrapper
```

Comment:
@let4be Yes, I here is quick explanation from comments

But I can give more deeply explanation:

`angular-co` triggers angular promise ($q) on co promise resolving/rejecting and it will be never resolve until we actually $scop.$digest do. But because I can't get inside of `angular-co` wrapper in tests to force angular's promise to resolve I just launch dirty update on some interval. By default it is 1s.